### ISSUE TITLE
Restrict copy-pasting same addresses in send form

### ISSUE THREAD

**Issue by shmoloko** [GMorDIE/gmordie#34]
It is possible to copy paste same address spamming ctrl+v in send form, with that it's easy to game leaderboard sending gm/Gn in batches of up to 589to your second account. 

![image](https://user-images.githubusercontent.com/76692450/189797730-c1386bea-a066-4fed-ac67-7bec6a61943b.jpeg)

![image](https://user-images.githubusercontent.com/76692450/189797746-046d83ff-c9cb-473c-a43d-0c553a59768c.jpeg)

Video in the link below https://discord.com/channels/978703103782174750/995998148113727548/1018750926548389898

**Comment by Tbaut** [comment]
I'd say we shouldn't prevent that, but rather count the amount of time, and send that amount of GN/GM instead of sending individual TXs.
Note that it's not because this UI prevents it, that I can't write a script, or modify the UI locally to allow me to do this.

**Comment by 0xKheops** [comment]
not sure what to think about this...as Tbaut said if chain allows it there is no reason to prevent it on the dapp.
@arrudagates wdyt ?

**Comment by arrudagates** [comment]
Yeah I don't think the dapp should disallow it, because then you just go to another dapp like p.js and do it there

**Comment by 0xKheops** [comment]
Got an evil plan to mitigate this :smiling_imp: 
ignoring duplicates (within a same block) from leaderboard total
